### PR TITLE
Feat/prepare for news site

### PR DESCRIPTION
**Pull request by jbpenrath**
- Add a news link on header and a section above projects
image

image

- Add a sentence to detail what is FUN
 
image

- Update course runs count

### CODE REVIEW

File: pages/_components/partials/nav.njk [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
+        <li><a class="nav-link scrollto" href="#about"><span>{{ 'section3' | i18n }}</span> </a></li>
+        <li><a class="nav-link scrollto" href="#funByNumber"><span>{{ 'section4' | i18n }}</span></a></li>
+        <li style="display:none;"><a class="nav-link scrollto" href="#usertest"><span>{{ 'section5' | i18n }}</span></a></li>
+        <li><a class="nav-link scrollto" href="#contact"><span>{{ 'section6' | i18n }}</span></a></li>
+        <li>
+            <a class="nav-link" href="{{ 'news_href' | i18n }}">
+                <i class="bi bi-box-arrow-up-right fs-4 me-3"></i>
+                {{ 'news' | i18n }}
+            </a>
+        </li>
```
Comment (NamFra) [inline pages/_components/partials/nav.njk:16]: FMI, is there a reason to place the link to the news site after the "contact us" section? So as not to disrupt navigation between sections?

Comment (jbpenrath) [reply pages/_components/partials/nav.njk:16]: Oh no, it does not affect navigation between sections at all but this is the only links which is an external link to it seems logic to myself that this link appears at the end of the list.

The only things required to navigate to an element is to fill the `href` attribute with the id of the targeted element.
e.g:
```html
...
<section id="section-1">...</section>
...
```

To navigate to the section `section-1`, you just have to create a link with href fullfilled with the targeted section id prefixed by a `#`
```
<a href="#section-1">Go to Section 1</a>
```

Comment (NamFra) [reply pages/_components/partials/nav.njk:16]: Ah yes! I misspoke. I meant that putting a link in the middle of the sections could be confusing for the user navigating through the menu. Which you confirmed ;) Everything is clear to me now, thanks!
The PR seems fine but of course I let Samuel approve it.

Comment (sampaccoud) [reply pages/_components/partials/nav.njk:16]: since it's external I would not include it here...

File: src/css/templates/_nav.scss [hunk 2]
```diff
@@ -194,6 +194,11 @@
     top: 15px;
     right: 15px;
   }
+
+  & > a.btn-emphasized.d-lg-block {
+    display: none !important;
+  }
```
Comment (sampaccoud) [inline src/css/templates/_nav.scss:200]: I didn't get this :man_shrugging:

Comment (jbpenrath) [reply src/css/templates/_nav.scss:200]: The `.d-lg-block` class does not leave choice.
Furthermore in this case, I though that is an acceptable solution as this is a specific element in a specific context so the impact here is really limited. (On desktop, when the menu is open, we hide the button).

But as a workaround, I could write myself the media query to show/hide button and link according to the viewport width.

File: pages/_components/partials/nav.njk (hunk 1 above)
Comment (jbpenrath) [reply pages/_components/partials/nav.njk:16]: But on mobile device, display the button next to the burger menu is not convenient.
Furthermore, as this is a subdomain, this is not exactly an external link (bad faith 😅)

Comment (sampaccoud) [reply pages/_components/partials/nav.njk:16]: IC

**Review by sampaccoud — APPROVED** [review]

**Pull request merged**